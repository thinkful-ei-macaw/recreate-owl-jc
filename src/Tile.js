import React from 'react'

const Tile = (props) => {

  return (
  <div>
    <img className="tile-image" alt="Avatar" src={props.avatar}/>
    <p>{props.name}</p>
  </div>
  );
  }
  
  export default Tile;