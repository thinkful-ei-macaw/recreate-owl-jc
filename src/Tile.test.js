import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Tile from './Tile';

describe('Tile component tests', () => {
  // smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Tile name="Koren Templeton" avatar="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"/>, div)
    ReactDOM.unmountComponentAtNode(div);
  })
  // snapshot test
  it('renders Tile with correct data', () => {
    const tree = renderer
    .create(<Tile name="Koren Templeton" avatar="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
  })
})