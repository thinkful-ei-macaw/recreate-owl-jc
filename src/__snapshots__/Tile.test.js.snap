// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile component tests renders Tile with correct data 1`] = `
<div>
  <img
    alt="Avatar"
    className="tile-image"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <p>
    Koren Templeton
  </p>
</div>
`;
